lib/matplotlib/tests/test_text.py

```diff
@@ -653,3 +653,11 @@ def test_buffer_size(fig_test, fig_ref):
     ax = fig_ref.add_subplot()
     ax.set_yticks([0, 1])
     ax.set_yticklabels(["€", ""])
+
+
+def test_fontproperties_presedence():
```

Comment:
```suggestion
def test_fontproperties_kwarg_precedence():
    """Test that kwargs take precedence over fontproperties defaults.“““
```
To make the purpose of the test clearer.